core/src/components/range/range.ionic.scss

```
@@ -0,0 +1,216 @@
+@use "../../themes/ionic/ionic.globals.scss" as globals;
+@import "./range.common";
+
+// Ionic Range
+// --------------------------------------------------
+
+:host {
+  --knob-border-radius: 50%;
+  --knob-background: #{globals.$ion-primitives-base-white};
+  --knob-box-shadow: 0;
+  --knob-size: 22px;
+  --knob-handle-size: #{globals.$ion-scale-1000};
+  --bar-height: #{globals.$ion-scale-200};
+  --bar-background: #{globals.$ion-primitives-neutral-100};
+  --bar-background-active: #{globals.ion-color(primary, base)};
+  --bar-border-radius: #{globals.$ion-border-radius-400};
+  --height: 42px;
```

I think this should use `scale-1000` to match the knob area and then we can adjust the pin in the other PR to reach the `68px` total height

```suggestion
  --height: #{globals.$ion-scale-1000};
```